code/modules/cargo/exports/materials.dm

```diff
@@ -22,8 +22,8 @@
 	if(istype(I, /obj/item/stack))
 		var/obj/item/stack/S = I
 		amount *= S.amount
-	else if(istype(I, /obj/item/ore))
-		amount *= 0.8 // Station's ore redemption equipment is really goddamn good.
+		if(istype(I, /obj/item/stack/ore))
+			amount *= 0.8 // Station's ore redemption equipment is really goddamn good.
```

Comment:
for a total of `amount * S.amount * 0.8` ?

Reply:
yes, per piece * pieces in stack * ore multiplier
unless it doesn't work this way, kek.